src/riak_core_handoff_cli.erl

```diff
@@ -146,11 +146,11 @@ details_usage() ->
 handoff_config(_Args, Flags) when length(Flags) > 1 ->
     [clique_status:text("Can't specify both --all and --node flags")];
 handoff_config(_Args, []) ->
-    clique_config:show(config_vars());
+    clique_config:show(config_vars(), []);
 handoff_config(_Args, [{all, _}]) ->
-    clique_config:show(config_vars() ++ ["-a"]);
+    clique_config:show(config_vars(), [{all, undefined}]);
```

Comment:
Ah, yes, fair point. Right now we don't support values for the --all flag, but it is theoretically possible we might in the future. I'll rebase in that change in just a moment...